mailpoet/RoboFile.php

```diff
@@ -1116,6 +1119,17 @@ public function releasePublishSlack($version = null) {
     $this->say("Release '$version[name]' info was published on Slack.");
   }
 
+  public function releaseRerunCircleWorkflow(string $project = null) {
+    $circleciController = $this->createCircleCiController();
+    $result = $circleciController->rerunLatestWorkflow($project);
+    // Sometimes can be useful to know which Circle project workflow was restarted
+    $project = $project ? " for the project '{$project}'" : '';
+    if (!$result) {
+      $this->yell("Circle Workflow{$project} was not restarted", 40, 'red');
+    }
+    $this->say("Circle Workflow{$project} was started from the beginning");
```

Comment:
I guess we need to put this line inside an else condition or finish the execution of this method when yell() is called to avoid having both the error and the success message being printed when there is an error?

Reply:
Good catch! I fixed it in the latest commit.

Reply:
Thanks